article/sections/_4_graph_construction.tex

```diff
@@ -1,3 +1,90 @@
-\section{Causal Graph Construction}
+\section{(Initial) Causal Graph Construction}
+\label{sec:graph-construction}
 
-\blindtext[2]
\ No newline at end of file
+Construction of an initial causal graph typically proceeds as follows.
+At a high level, we
+\begin{enumerate}
+   \item adopt a population perspective,
+   \item brainstorm all variables that we think affect the system that generates our observations,
+   \item remove any variables that would cause bias in our causal inferences,
+   \item connect all variables in our graph according to our a-priori beliefs about causal relations amongst these variables,
+   \item consider how the graph structure may differ between individuals and subgroups within the population.
+\end{enumerate}
+The following paragraphs will describe these steps in detail.
+
+To begin, we adopt the position of a researcher concerned about population level relationships.
+This means we will think through what is a likely generative model for all individuals.
+Later on, we will devote time to thinking about how subgroups and individual heterogeneity may affect our causal graphs.
```

Comment:
Similar question as  elsewhere: what sort of affects from subgroups and individual heterogeneity are you thinking of that don't result in a different causal graph?

Also, I changed the beginning of the sentence to say "Later in this section [...]". I didn't want to go with the suggested edit because it makes the sentence preamble appreciably longer with large words, and it would divert the sentence's focus from what will go on later to more of what happened before.